docs/troubleshooting_deis/troubleshooting-store.rst

```diff
@@ -0,0 +1,124 @@
+:title: Troubleshooting deis-store
+:description: Resolutions for common issues with deis-store and Ceph.
+
+.. _troubleshooting-store:
+
+Troubleshooting deis-store
+==========================
+
+The store component is the most complex component of Deis. As such, there are many ways for it to fail.
+Recall that the store components represent Ceph services as follows:
+
+* ``store-monitor``: http://ceph.com/docs/giant/man/8/ceph-mon/
+* ``store-daemon``: http://ceph.com/docs/giant/man/8/ceph-osd/
+* ``store-gateway``: http://ceph.com/docs/giant/radosgw/
+* ``store-metadata``: http://ceph.com/docs/giant/man/8/ceph-mds/
+* ``store-volume``: a system service which mounts a `Ceph FS`_ volume to be used by the controller and logger components
+
+Log output for store components can be viewed with ``deisctl status store-<component>`` (such as
+``deisctl status store-volume``). Additionally, the Ceph health can be queried by using the ``deis-store-admin``
+administrative container to access the cluster.
+
+.. _using-store-admin:
+
+Using store-admin
+-----------------
+
+``deis-store-admin`` is an optional component that is helpful when diagnosing problems with ``deis-store``.
+It contains the ``ceph`` client and writes the necessary Ceph configuration files so it always has the
+most up-to-date configuration for the cluster.
+
+To use ``deis-store-admin``, install and start it with ``deisctl``:
+
+.. code-block:: console
+
+    $ deisctl install store-admin
```

Comment:
following up from my last comment about refresh-units, this will result in an error unless they already have downloaded the unit to their filesystem themselves, so we either need to add documentation around that or toss it into refresh-units.